recipe/conda_build_config.yaml

```
@@ -0,0 +1,5 @@
+channel_targets:
+  - conda-forge zlib-ng-compat-test
```

Put non compat builds in main.
```suggestion
zip_keys:
  - - channel_targets
    - compat
channel_targets:
  - conda-forge zlib-ng-compat-test
  - conda-forge main
```